src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import React from 'react';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
```

Comment:
As I mentioned above, here you are doing the same iteration in tabs array, see?
This is an indicative that the whole tab object could be passed to this component and everything would be solved in the parent component